src/utils/cli.rs

```diff
@@ -55,6 +55,9 @@ enum Subcommands {
         #[arg(short, long, default_value_t = false)]
         /// Serve an individual stele instead of the Stele specified in config.toml.
         individual: bool,
+        /// Bind to specify ip address instead of 127.0.0.1
+        #[arg(short, long, default_value = "127.0.0.1")]
+        bind_to: String,
```

Comment:
could we do String::from here and avoid the clippy pattern type mismatch lint issue?
```suggestion
        #[arg(short, long, default_value = String::from("127.0.0.1"))]
        bind_to: String,
```